packages/pages/lib/landing/templates/roadmap.js

```diff
@@ -1,14 +1,22 @@
 const html = require('../../html/html');
 
 function renderMilestones(milestones) {
+	let hasCurrent = false;
+	let hasFuture = false;
 	let output = "";
 	milestones.forEach((milestone, index) => {
-		if (index > 0 && milestones[index-1].type !== milestone.type) {
-			output += '<h4 class="roadmap__future">Future Milestones</h4>';
+		if ('current' === milestone.type && !hasCurrent) {
```

Comment:
maybe more interesting to highlight where we are?
Or highlight both in a different style?